test/rubygems/test_gem_uninstaller.rb

```diff
@@ -295,8 +295,15 @@ def test_uninstall_default_gem_with_same_version
 
     uninstaller = Gem::Uninstaller.new spec.name, :executables => true
 
-    uninstaller.uninstall
-
+    ui = Gem::MockGemUi.new "1\ny\n"
+    use_ui ui do
+      uninstaller.uninstall
+    end
+    expected = "Successfully uninstalled default-2\n" \
+      "There was both a regular copy and a default copy of default-2. The " \
```

Comment:
Do we use "copy" term at all in RubyGems?

Reply:
"Installation" maybe? This is a very edge case situation, so the term is probably not used anywhere else, indeed.

Reply:
What do you think @simi? Should we use "installation" or another alternative, or is "copy" ok even if it's not used anywhere else?

Reply:
Alright, let's merge this and we can further tweak things as followup PR's.